selectors.yml

```
@@ -44,6 +62,9 @@ selectors:
   84532:
     selector: 10344971235874465080
     name: "ethereum-testnet-sepolia-base-1"
+  10200: 
+    selector: 8871595565390010547
+    name: "gnosis-testnet-chiado"
```

Please, refer to [prior art](https://github.com/smartcontractkit/atlas/tree/master/postgres/migrations/evm/gnosis_chain-testnet-chiado) when addint any network which some product may already support!
```suggestion
    name: "gnosis_chain-testnet-chiado"
```